applications/common/backend/src/main/resources/libs/kyanite/common/components/title/title.html

```diff
@@ -24,6 +24,18 @@ <h6 class="has-text-primary is-1 eyebrow">
   </sly>
   <h1 id="${model.anchorId}" style="text-align: ${model.titleAlign @ context='styleToken'};" data-sly-element="${model.element}" class="${model.titleClasses @ join=' '}">
     ${model.textAsOneLine @ context='html'}
+    <sly data-sly-test="${model.addAnimatedEndings}">
+      <span id="typed-strings" class="${model.endingsClasses @ join=' '}">
+        <sly data-sly-list.item="${model.endings}">
+          <p>
```

Comment:
Should we use a paragraph inside the heading? Its direct parent is a span element, but I still don't know if it's correct.

Reply:
yeah, it is not correct, changed to <span>